### PR TITLE
refactor: egctl cmds

### PR DESCRIPTION
**Pull request by qicz**

### CODE REVIEW

File: internal/cmd/egctl/bootstrap_config.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// Copyright Envoy Gateway Authors
```
Comment (arkodg) [inline internal/cmd/egctl/bootstrap_config.go:1]: shouldnt it be `config_bootstrap.go` ?

Comment (qicz) [reply internal/cmd/egctl/bootstrap_config.go]: chanaged

**Comment by qicz** [comment]
@arkodg  pls review again. ci fixed same with #1168 ,  remove this fix?

**Review by arkodg — APPROVED** [review]

**Review by Xunzhuo — APPROVED** [review]
Thanks @qicz

**Pull request merged**